app/service/EmailService.scala

```diff
@@ -19,23 +19,30 @@ package service
 import javax.inject.{Inject, Singleton}
 import uk.gov.hmrc.http.HeaderCarrier
 import connector.EmailConnector
-import models.{Case, CaseCompletedEmail, CaseCompletedEmailParameters, EmailTemplate}
+import models.{Case, CaseCompletedEmail, CaseCompletedEmailParameters, EmailTemplate, Operator}
 import utils.JsonFormatters.{emailCompleteParamsFormat, emailFormat}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
+import utils.Dates
 
 @Singleton
 class EmailService @Inject() (connector: EmailConnector) {
 
-  def sendCaseCompleteEmail(c: Case)(implicit hc: HeaderCarrier): Future[EmailTemplate] = {
+  def sendCaseCompleteEmail(c: Case, operator: Operator)(implicit hc: HeaderCarrier): Future[EmailTemplate] = {
     if (!c.application.isBTI) {
       throw new IllegalArgumentException("Cannot send email for non BTI types")
     }
 
     val email: CaseCompletedEmail = CaseCompletedEmail(
       Seq(c.application.contact.email),
-      CaseCompletedEmailParameters(c.application.contact.name, c.reference, c.application.asATAR.goodName)
+      CaseCompletedEmailParameters(
+        recipientName_line1 = c.application.contact.name,
+        reference = c.reference,
+        goodsName = c.application.asATAR.goodName,
+        officerName = operator.name.getOrElse(""),
```

Comment:
There is no good alternative that we can use here. It will render as an empty line in the email, i.e. something like `Yours sincerely,   Tariff Classification Team`